gui.js

```diff
@@ -3818,9 +3818,11 @@ var CGUI = function()
     // Set up GUI elements
     document.getElementById("osc1_vol").sliderProps = { min: 0, max: 255 };
     document.getElementById("osc1_semi").sliderProps = { min: 92, max: 164 };
+    document.getElementById("osc1_xenv").sliderProps = { min: 0, max: 64 };
```

Comment:
It seems to me that we could increase this range, and increase the granularity. E.g. `max: 255` and then divide the by 16 instead of 4. There might be a size gain if we only divide by a single digit figure, e.g. 8 or 9, though.